.github/workflows/container.yml

```diff
@@ -31,3 +31,26 @@ jobs:
           push: ${{ github.event_name == 'push' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  build-search:
+    name: Build search
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - uses: docker/metadata-action@v5
+        id: meta
+        with:
+          images: |
+            ghcr.io/${{ github.repository_owner }}/search
+      - uses: actions/checkout@v4
+      - uses: docker/build-push-action@v6
+        with:
+          file: Dockerfile.search
+          # push: ${{ github.event_name == 'push' }}
```

Comment:
I'm not familiar enough with github actions... what does hard coding true do here vs your event above?